python/CMakeLists.txt

```diff
@@ -75,22 +58,15 @@ include(CheckCXXCompilerFlag)
 check_cxx_compiler_flag("-Wall -Werror -pedantic" HAVE_PEDANTIC)
 
 if(HAVE_PEDANTIC)
-  target_compile_options(cpp PRIVATE -Wall;-Werror;-pedantic)
-endif()
-
-# In Debug mode override pybind11 symbols visibility Symbols must be visible to
-# backtrace_symbols() to produce nice logs
-if(CMAKE_BUILD_TYPE STREQUAL "Debug")
-  target_compile_options(cpp PRIVATE "-fvisibility=default")
+  # target_compile_options(cpp PRIVATE -Wall;-Werror;-pedantic)
```

Comment:
Leftover comment. I wonder how does symbols visibility (and readable backtrace) work with nanobind?

Reply:
I'll leave this in because we'd like to turn it on again. Need to figure out how to use `isystem` for the nanobind headers.